src/EditorFeatures/Test2/Expansion/NameExpansionTests.vb

```diff
@@ -222,7 +222,7 @@ class C
             Await TestAsync(input, expected, expandParameter:=True)
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Expansion)>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/11979"), Trait(Traits.Feature, Traits.Features.Expansion)>
```

Comment:
Please disregard this comment if your change caused the test failure. In that case, however, I think we should have a plan for fixing this behavior. If it is not too hard to do, I suggest to fix it as part of this PR.

---
In reply to: [67015720](https://github.com/dotnet/roslyn/pull/11981#discussion_r67015720) [](ancestors = 67015720)